### ISSUE TITLE
BackstopJS issue  - same image with different screen size

### ISSUE THREAD

**Issue by VinnyIla** [garris/BackstopJS#915]
I am an Automation Engineer, using your BackstopJS for visual regression testing.
I have been using this for a while and was quite impressed with its results. The scrubber functionality in the reporting makes it much useful to know where it failed comparatively.

Recently I started noticing that, the same browser size is launched differently, when TeamCity agent is set to auto login and start. Because of this, the test image is slightly different in dimensions to the reference image.
I do set "RequireSameDimensions" to false. Is it possible for me to ignore screen size and just compare the content.
 
If you could provide me with some suggestions to resolve this, that would be much appreciated.

diff%: 4.18 diff-x: -636 diff-y: -257  

Reference Image:
![image](https://user-images.githubusercontent.com/34079043/48383572-6dbca880-e73a-11e8-8917-dcefbe302ebc.png)

Test Image:

![image](https://user-images.githubusercontent.com/34079043/48383596-862cc300-e73a-11e8-8b7f-4f8725d264c9.png)

Diff Image:
![image](https://user-images.githubusercontent.com/34079043/48383607-8dec6780-e73a-11e8-925f-3206e0190a89.png)

**Comment by VinnyIla** [comment]
Yes, but i need to compare the screen as a whole. There are more than 50 screens like that we use for testing.

**Comment by garris** [comment]
I am not sure what is causing your screen size change but there are probably many many ways to address this.  Maybe you could set an absolute width on your container when onReady runs?  There is often no "right" way to address this kind of thing -- allow yourself to be creative.   Good luck.

**Comment by rostislaw9** [comment]
Hello. Has this issue been diagnosed?

I've started using your _BackstopJS_ recently and now am experiencing the same issue. I couldn't find the solution online so I tried to figure out what exactly causes the problem myself, but to no avail.

I'd like to note that it's not always that the same pair of images are of a different size, but sometimes they both are of a wrong size (which differs from those specified in the config). Thus, it could show successful tests results, but in fact target resolutions are not tested completely.

With this config:
```
"viewports": [
      {
        "label": "phone1",
        "width": 320,
        "height": 480
      }
    ]
```
Some results are ok:
![Screenshot from 2020-01-17 10-43-42](https://user-images.githubusercontent.com/51000328/72587270-77313a00-3916-11ea-8670-571955f60d30.png)
Some fail with different size:
![Screenshot from 2020-01-17 10-44-01](https://user-images.githubusercontent.com/51000328/72587271-77313a00-3916-11ea-8e26-77fa4a51b6ea.png)

**Comment by rostislaw9** [comment]
Worked this out by using `"selectors": ["html"]` to capture full page screenshots instead of using `"selectors": ["document"]`
